src/box/sql/mem.h

```diff
@@ -142,6 +142,18 @@ mem_is_num(const struct Mem *mem)
 			MEM_TYPE_DEC)) != 0;
 }
 
+static inline bool
+mem_is_any(const struct Mem *mem)
+{
+	return (mem->flags & MEM_Any) != 0;
+}
+
+static inline bool
+mem_is_msgpack(const struct Mem *mem)
```

Comment:
I would recommend to call it `mem_is_dict` or likewise. In future they might be stored not as msgpack, or some new type would be both msgpack and scalar.

Reply:
I used `mem_is_container` since I already changed name of test to `container_test.lua` from `msgpack.test.lua`.